platform/ui-next/src/components/InvestigationalUseDialog/InvestigationalUseDialog.tsx

```diff
@@ -68,15 +70,15 @@ const InvestigationalUseDialog = ({
           <Icons.InvestigationalUse className="h-18 w-18" />
           <div className="flex flex-col">
             <div className="text-[19px] text-white">
-              OHIF Viewer is{' '}
-              <span className="text-primary-light">for investigational use only</span>
+              OHIF Viewer{' '}
+              <span className="text-primary-light">{t('is for investigational use only')}</span>
```

Comment:
why are you separating the is?

Reply:
In other languages "is" will be displayed regardless of the actual selected language. "Is" is an English language word is not it? :)

When we translate this sentence (OHIF Viewer is for investigational use only) to another language we leave intact only the application name "OHIF Viewer", otherwise we'll get "OHIF Viewer is только для исследовательских целей."

If I understand the translation correctly we should extract all language specific words to translation files, and "is" does not belong to the application name (OHIF Viewer) and in different languages it will look in a different way.

In case my explanation is not clear feel free to ask for clarification.

Reply:
Don't change this one. Do like `t('is for investigationalxxxxx') || t('for investigational')`